docs/ProductDescription.md

```diff
@@ -39,6 +39,9 @@ This Component can be imported and used by any VTEX App.
 | `collapseContent` | `Boolean` | If true, whenever the product description is too big, it will collapse and show a "Show More" button. When false, it will never collapse and will always show the whole description. | `true` |
 | `title`           | `string`  | Defines a custom title for the description section. | `undefined` |
 | `classes` | `CustomCSSClasses` | Used to override default CSS handles. To better understand how this prop works, we recommend reading about it [here](https://github.com/vtex-apps/css-handles#usecustomclasses). Note that this is only useful if you're using this block as a React component. | `undefined` |
+| `customCollapse`           | `boolean` | Overrides the `collapseContent` prop and allows you to collapse content by number of characters. | `false` |
+| `desktopLimitCharacters`   | `number`  | Works only if `collapseContent` is `true`. It determine the number of characters shown for desktop devices. | `1000` |
+| `mobileLimitCharacters`    | `number`  | Works only if `collapseContent` is `true`. It determine the number of characters shown for mobile devices. | `500` |
```

Comment:
I think it's better to create a prop called `ellipsisContent`. This prop would be a [responsive-value prop](https://github.com/vtex-apps/responsive-values).

So the user could do something like:

```tsx
"ellipsisContent": {
  "mobile": 500,
  "desktop": 1000
}
  ```

Or if it's the same value for both:

```tsx
"ellipsisContent": 600
```

If `ellipsisContent` is set, we can consider it's active.

Reply:
I agree with that. What do you think about @khrizzcristian ?